### DISCUSSION TITLE
Teleconference Possible Bug

### DISCUSSION THREAD

**Discussion by rcadena** [leewoodridge/RenegadeBBS#4] [General]
Having not had a multimode bbs before, nor used RG teleconference much, I might be off about how this is supposed to work.

Summary: the /U (show users in room) doesn't show anything if there is more than one user in the room.  
The way I expect it to work, of course, is that any user in the room should show up in the list.

Here's a video of how it's working on two nodes, compiled on the HEAD of main.
![teleconf-users](https://github.com/user-attachments/assets/19f3c55c-407c-47eb-b2c0-018600062bbd)

The [fix is simple](https://github.com/leewoodridge/RenegadeBBS/compare/main...rcadena:RenegadeBBS:teleconf-show-users?expand=1): the check in ShowRoom only goes into the loop to show the users if `People = 0`, so just make it check for not zero. 

Here's my branch with the fix.
![teleconf-fix](https://github.com/user-attachments/assets/a4d8771d-2669-44ef-b70e-7e5bc1d78705)

**Comment by Renegade-Exodus** [comment]
Added this to the main branch.  Thanks.  We we tinkering with the
teleconference a few years back to make it look more like an IRC chat room,
but then gave up as no on seemed to use it.  Feel free to keep sending
those fixes you spot!  Thanks!

**Reply by rcadena** [reply]
cool. I noticed that it's also not quite working for /S - I'll poke around. probably similarly easy fix.

**Reply by rcadena** [reply]
actually, I guess it does, but it got kinda rewrite on top of the previous text. Ignore my previous comment.

I think the way stuff gets displayed might be a little funky, so I might mess around to see if it's my text resolution or just the ui is not moving stuff around correctly.
